components/system_notice/notices.jsx

```diff
@@ -50,5 +59,48 @@ export default [
                 defaultMessage='Some policy and permission System Console settings have moved with the release of <a href="https://about.mattermost.com/default-advanced-permissions" target="_blank">advanced permissions</a> in Enterprise E10 and E20.'
             />
         ),
+        cantForget: false,
+        show: (serverVersion, config, license) => {
+            if (license.IsLicensed === 'false') {
```

Comment:
These methods basically amount to selectors. I recall Harrison recently proposing that we avoid passing in large structs just to compute a few small things, and instead hoist new selectors for the component directly in the connector. Not sure if that's feasible here, but thought I'd raise it just in case.

Reply:
Yes, but in this case is about the possibility of perform complex checks to show the notices. I understand the concern, but I don't see a big deal in this case.